index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+  <main id="main">
+    <h1 id="title">Diego Armando Maradona</h1>
+    <section id="img-div">
+      <img id="image" src="https://i.pinimg.com/originals/79/85/08/7985080c1431619fa89314e11aff86b3.png" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+      <h5 id="img-caption"> 
+        This image represent Diago Armando Maradona making some magic things with the ball!
+      </h5>
+    </section>
+    <section id="tribute-info">
+      <h3 id="headline">Here's a time line of Maradona's life:</h3>
+      <ul>
+          <li><strong>1971</strong> Scouted age 11 for the junior team of Argentinos Juniors</li>
+          <li>
+            <strong>1976</strong> Makes his professional debut aged 15
+          </li>
+          <li>
+            <strong>1977</strong> Scores first international goal and wins the junior World Cup
+          </li>
+          <li>
+            <strong>1981</strong> Transfers to Boca Juniors for $1.96m.
+          </li>
+          <li>
+            <strong>1982</strong> Plays his first World Cup for Argentina, scoring twice, but also getting sent off. Afterwards he is transferred to Barcelona FC for a then world record $9.81m.
+          </li>
+          <li>
+            <strong>1983</strong>  Wins the Spanish Cup with Barcelona         </li>
+          <li>
+            <strong>1984</strong>  Transfers from Barcelona to Serie A Napoli for another record fee of $13.54m.
+          </li>
+          <li>
+            <strong>1986</strong>  Captains Argentina to World Cup Victory, scoring two goals against England on the way, the first, the infamous "Hand of God" and the second an incredible 50 meter run voted "Goal of the Century" in a 2002 FIFA poll.
+          </li>
+          <li>
+            <strong>1987</strong>  Helps Napoli to their first Italian title
+          </li>
+          <li>
+            <strong>2000</strong> Collapses with heart problems
+          </li>
+          <li>
+            <strong>2020</strong> Maradona dies in his house.
+          </li>
+      </ul>
+      <blockquote cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
+          <p>
+            "Maradona was a great football player, he won the 1986 World Cup with Argentina Football Team. He won 2 times the Seria-A ligue with Napoli football club."
+          </p>
```

Comment:
- [ ] OPTIONAL: You could separate in differents lines your p text in order to be more clean and easy to read what is inside!